docs/nnx/index.rst

```diff
@@ -8,11 +8,15 @@ NNX
 
 ----
 
-NNX is a new Flax API that is designed to make it easier to create, inspect, debug,
-and analyze neural networks in JAX. It achieves this by adding first class support
+Flax NNX is a new simplified API that is designed to make it easier to create, inspect,
+debug, and analyze neural networks in JAX. It achieves this by adding first class support
 for Python reference semantics, allowing users to express their models using regular
-Python objects. NNX takes years of feedback from Linen and brings to Flax a simpler
-and more user-friendly experience.
+Python objects. Flax NNX takes years of feedback from Flax Linen to bring a simpler and more
```

Comment:
Here I would stress the continuity rather than the break:
"Flax NNX is an evolution of the previous Flax Linen APIs building upon years of experience to bring ..."

I would also say that Flax NNX and Flax Linen can interoperate.